### ISSUE TITLE
No longer working in 2021?

### ISSUE THREAD

**Issue by JSKF** [jackpark071801/yuumibot#1]
From my own usage, ive found Yuumi constantly recalling in fountain, I was to try this out in some custom games with friends but it just plain wont work. is there something i'm missing here? the program starts fine it just wont play the game correctly.

**Comment by jackpark071801** [comment]
Yeah I’m so sorry but the bot only worked in season 10 (2020) and no longer
works right now with season 11.

**Comment by sapkodas** [comment]
Hi, I'm currently trying to bring this bot back to life, can you explain to me on what the findWhite function does? Thanks
